wiki/Tournaments/UKCC/7/en.md

```diff
@@ -78,7 +78,7 @@ This competition has come to an end and resulted in the following podium:
 | :-: | :-- |
 | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | [Bubbleman](https://osu.ppy.sh/users/5182050) |
 | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | [Spare](https://osu.ppy.sh/users/2204373) |
-| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | [AJT](https://osu.ppy.sh/users/3181083) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | [hlo](https://osu.ppy.sh/users/3181083) |
```

Comment:
Resolved with https://github.com/ppy/osu-wiki/pull/6341/commits/682662be8ff6091e687d573febf751a9eece6010

Purely a mistake on my part, should be correct now. AJT was chosen over Nega since that is what they finished the tournament with and is still their current username.